packages/page-accounts/src/modals/Qr.tsx

```diff
@@ -55,19 +56,23 @@ function QrModal ({ className = '', onClose, onStatusChange }: Props): React.Rea
 
   const _onScan = useCallback(
     (scanned: Scanned): void => {
-      setAddress({
-        address: scanned.isAddress
-          ? scanned.content
-          : keyring.createFromUri(scanned.content, {}, 'sr25519').address,
-        isAddress: scanned.isAddress,
-        scanned
-      });
+      if (isEthereum && scanned.isAddress && scanned.content.substring(0, 2) !== '0x') {
+        setAddress({ address: '', isAddress: false, scanned: null, warning: 'Ethereum Compatible Parachains must import from an Ethereum Wallet' });
+      } else {
+        setAddress({
+          address: scanned.isAddress
+            ? scanned.content
+            : keyring.createFromUri(scanned.content, {}, 'sr25519').address,
+          isAddress: scanned.isAddress,
+          scanned
+        });
+      }
 
       if (scanned.name) {
         _onNameChange(scanned.name);
       }
     },
-    [_onNameChange]
+    [_onNameChange, isEthereum]
```

Comment:
```suggestion
    [_onNameChange, isEthereum, t]
```

Since we use `t` above for the warning string